web-playground/e2e/sample-tester.test.ts

```diff
@@ -0,0 +1,80 @@
+import { FrameWaitForFunctionOptions, Page } from "puppeteer";
+import * as fs from 'fs';
+import * as path from 'path';
+
+const TimeoutMs = 20000;
+
+describe('test all grammars with puppeteer', () => {
+    beforeEach(async () => {
+        await page.goto('http://localhost:3000/playground');
+    });
+
+    it('can format', async () => {
+        const rootDir = path.join(__dirname, "../../");
+        const inputDir = path.join(rootDir, "topiary/tests/samples/input/");
+        const expectedDir = path.join(rootDir, "topiary/tests/samples/expected/");
+        const queryDir = path.join(rootDir, "languages/");
+
+        for (let inputFileName of await fs.promises.readdir(inputDir)) {
```

Comment:
We have a pending issue about tests breaking if there are unexpected files in the `input`/`expected` directories (#390).
Maybe we should mark this code location to be considered as well when this issue is addressed?